src/Elastic.CommonSchema.Serilog/EcsTextFormatterConfiguration.cs

```diff
@@ -19,6 +20,7 @@ public interface IEcsTextFormatterConfiguration
 		Func<Base, LogEvent, Base> MapCustom { get; set; }
 		bool MapExceptions { get; set; }
 		IHttpAdapter MapHttpAdapter { get; set; }
+		IList<string> LogEventPropertiesToFilter { get;set; }
```

Comment:
This might be better as an `ISet<string>` 

1. for fast lookup
2. to allow String comparison implementation to be defined by the user as part of the concrete `ISet<string>` implementation

Reply:
Excellent point! Updated with bbcb645502f950d8708911ea91aaed8ca19f06c7